Lib/test/test_os.py

```diff
@@ -234,17 +234,16 @@ def test_read(self):
             self.assertEqual(s, b"spam")
 
     @support.cpython_only
-    # Skip the test on 32-bit platforms: the number of bytes must fit in a
-    # Py_ssize_t type
-    @unittest.skipUnless(INT_MAX < PY_SSIZE_T_MAX,
-                         "needs INT_MAX < PY_SSIZE_T_MAX")
     @support.bigmemtest(size=INT_MAX + 10, memuse=1, dry_run=False)
```

Comment:
issue here is that this will result in a memory allocation error on 32 bit machines, and this bigmemtest effectively makes it not run on 32 bit machines (They are unlikely to have that much RAM)